### PR TITLE
Add session property for druid connector

### PR DESCRIPTION
**Pull request by zhenxiao**
```
== RELEASE NOTES ==

Druid Changes
* session property, druid.pushdown, it added, to control whether to pushdown all query processing to druid
```

### CODE REVIEW

File: presto-druid/src/main/java/com/facebook/presto/druid/DruidSessionProperties.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+import javax.inject.Inject;
+
+import java.util.List;
+
+import static com.facebook.presto.spi.session.PropertyMetadata.booleanProperty;
+
+public class DruidSessionProperties
+{
+    private static final String PUSHDOWN = "pushdown";
```
Comment (highker) [inline presto-druid/src/main/java/com/facebook/presto/druid/DruidSessionProperties.java:29]: compute_pushdown_enabled

File: presto-druid/src/main/java/com/facebook/presto/druid/DruidConfig.java [hunk 2]
```diff
@@ -65,4 +66,17 @@ public DruidConfig setDruidSchema(String schema)
         this.schema = schema;
         return this;
     }
+
+    public boolean getPushdown()
+    {
+        return pushdown;
+    }
+
+    @Config("druid.pushdown")
```
Comment (highker) [inline presto-druid/src/main/java/com/facebook/presto/druid/DruidConfig.java:75]: `druid.compute-pushdown-enabled`

**Comment by zhenxiao** [comment]
thank you, @highker 
get comments addressed

File: presto-druid/src/main/java/com/facebook/presto/druid/DruidConfig.java [hunk 3]
```diff
@@ -65,4 +66,17 @@ public DruidConfig setDruidSchema(String schema)
         this.schema = schema;
         return this;
     }
+
+    public boolean getPushdown()
```
Comment (highker) [inline presto-druid/src/main/java/com/facebook/presto/druid/DruidConfig.java:70]: `isComputePushdownEnabled`

File: presto-druid/src/main/java/com/facebook/presto/druid/DruidConfig.java [hunk 4]
```diff
@@ -65,4 +66,17 @@ public DruidConfig setDruidSchema(String schema)
...
+    {
+        return pushdown;
+    }
+
+    @Config("druid.compute-pushdown-enabled")
+    @ConfigDescription("pushdown query processing to druid")
+    public DruidConfig setPushdown(boolean pushdown)
```
Comment (highker) [inline presto-druid/src/main/java/com/facebook/presto/druid/DruidConfig.java:77]: setComputePushdownEnabled

File: presto-druid/src/main/java/com/facebook/presto/druid/DruidSessionProperties.java [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+import static com.facebook.presto.spi.session.PropertyMetadata.booleanProperty;
+
+public class DruidSessionProperties
+{
+    private static final String COMPUTE_PUSHDOWN = "compute_pushdown_enabled";
+
+    private final List<PropertyMetadata<?>> sessionProperties;
+
+    public static boolean isPushdown(ConnectorSession session)
```
Comment (highker) [inline presto-druid/src/main/java/com/facebook/presto/druid/DruidSessionProperties.java:33]: same

File: presto-druid/src/test/java/com/facebook/presto/druid/TestDruidConfig.java [hunk 6]
```diff
@@ -40,12 +41,14 @@ public void testExplicitPropertyMappings()
                 .put("druid.broker-url", "http://druid.broker:1234")
                 .put("druid.coordinator-url", "http://druid.coordinator:4321")
                 .put("druid.schema-name", "test")
+                .put("druid.pushdown", "true")
```
Comment (highker) [inline presto-druid/src/test/java/com/facebook/presto/druid/TestDruidConfig.java:44]: need to change as well

**Comment by zhenxiao** [comment]
thank you, @highker 
comments addressed

**Pull request merged**